user_management/api/tests/test_models.py

```diff
@@ -8,17 +8,17 @@ class TestAuthToken(utils.APIRequestTestCase):
     model = AuthToken
 
     def test_fields(self):
-        fields = self.model._meta.get_all_field_names()
+        fields = get_all_field_names(self.model)
 
-        expected = wipe_id_fields_on_django_lt_17((
+        expected = (
```

Comment:
As Django 1.7 is no longer supported, it might be worth completely removing the `wipe_id_fields_on_django_lt_17` helper method.